packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/JoinZoomButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { cn } from '@/app/utils/generalUtils'
+import { Button, ButtonProps, Tooltip } from 'antd'
+import { PhoneOutlined } from '@ant-design/icons'
+import { PropsWithChildren } from 'react'
+
+interface JoinZoomButtonProps extends ButtonProps {
+  zoomLink?: string
+  onJoin?: () => void
+}
+
+const JoinZoomButton: React.FC<PropsWithChildren<JoinZoomButtonProps>> = ({
```

Comment:
I noticed that there is also a JoinZoomButton in QueueInfoColumn. I feel one of these was meant to be deleted?

Reply:
No, that was intentional. I put it there because the pop up modal for joining a zoom call would close after you clicked the link. So if you accidentally lose the tab and need the link again, that button will remain in the info column until the student is no longer being helped.